src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -6,9 +6,9 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
          http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
 
-    <changeSet id="1" author="foxrev">
-        <createTable tableName="Scooter">
-            <column name="id" type="varchar(36)">
+    <changeSet id="3" author="foxrev">
```

Comment:
it doesn' t work this way 😅you should not increase the version of the same changeset, you create a new one when you modify a table.